feat: Russian update

larentoun opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
<!-- Include a small to medium description of what your PR changes. Document every changes or this may delay review or even discourage maintainers from merging your PR! -->
- Исправление и стандартизация перевода
- Перевод задач
- Перевод осмотра
- Мелкие правки перевода attacktext
- Частичный перевод Принца Террора, Коровы, Уники и Челюстей Жизни для тестов перевода
- Возвращение частоты сообщений шуток скелета
- Перенос дефайнов падежей в _DEFINES
- Перевод проков дальнобойного оружия

## Why It's Good For The Game
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
Русский язык на русскоязычном билде.

## Images of changes
<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif of your feature if you want -->

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

larentoun committed:
fix: Исправление перевода

larentoun committed:
feat: добавление перевода для задач

larentoun committed:
feat: Перевод для осмотра, осмотра оружия

larentoun committed:
feat: перевод attacktext

larentoun committed:
feat: мелкие правки перевода

larentoun committed:
feat: частичный перевод для тестов

larentoun committed:
feat: реверт спама шутками скелета

larentoun committed:
feat: Перенос дефайнов

larentoun committed:
feat: Перевод огнестрела

larentoun commented on code/game/objects/items/tools/crowbar.dm:
```diff
@@ -62,15 +62,17 @@
 	name = "jaws of life"
-	desc = "A set of jaws of life, the magic of science has managed to fit it down into a device small enough to fit in a tool belt. It's fitted with a prying head."
+	ru_names = list(NOMINATIVE = "челюсти жизни", GENITIVE = "челюстей жизни", DATIVE = "челюстям жизни", ACCUSATIVE = "челюсти жизни", INSTRUMENTAL = "челюстями жизни", PREPOSITIONAL = "челюстях жизни")
+	desc = "Это набор челюстей жизни, и магия науки позволила нам поместить это в достаточно маленькое устройство, чтобы оно вместилось в пояс для инструментов. Установлена выпирающая головка"
```
Да и вообще, оставить их так или в "гидравлические" перевести?

PhantornRU replied:
Челюсти Жизни - это название, лучше их так и оставить.

VasilevAY replied:
Лучше так 
Это - "Челюсти Жизни", и магия....

"набор челюстей жизни" - выглядит и звучит плохо

PhantornRU replied:
согласен

larentoun commented on code/game/objects/items/tools/wirecutters.dm:
```diff
@@ -70,7 +70,9 @@
 	name = "jaws of life"
-	desc = "A set of jaws of life, the magic of science has managed to fit it down into a device small enough to fit in a tool belt. It's fitted with a cutting head."
+	ru_names = list(NOMINATIVE = "челюсти жизни", GENITIVE = "челюстей жизни", DATIVE = "челюстям жизни", ACCUSATIVE = "челюсти жизни", INSTRUMENTAL = "челюстями жизни", PREPOSITIONAL = "челюстях жизни")
+	desc = "Это набор челюстей жизни, и магия науки позволила нам поместить это в достаточно маленькое устройство, чтобы оно вместилось в пояс для инструментов. Установлена разрезающая головка"
```
.

larentoun commented on code/game/objects/items/toys.dm:
```diff
@@ -1520,8 +1520,8 @@
-	. += "Has [fake_bullets] round\s remaining."
-	. += "[fake_bullets] of those are live rounds."
+	. +=  "Боезапаса внутри: [fake_bullets]"
+	. +=  "Из них заряжено: [fake_bullets]"
```
Есть лучше идеи?

PhantornRU replied:
Патронтаж

larentoun replied:
"Боезапаса внутри:" сменить на "Патронтаж:"?

PhantornRU replied:
пока можно оставить так как есть и оставить мозгоебку для потомков. СБ-шники переводчики думаю когда-нибудь да поправят как им самим хочется и напишут падежные системы.

larentoun commented on code/modules/mob/living/simple_animal/bot/bot.dm:
```diff
@@ -279,7 +279,7 @@
 	user.do_attack_animation(src)
 	apply_damage(rand(15,30), BRUTE)
-	visible_message("<span class='danger'>[user] has slashed [src]!</span>")
+	visible_message("<span class='danger'>[user.declent_ru(NOMINATIVE)] полоснул [src.declent_ru(ACCUSATIVE)]!</span>")
```
.

larentoun commented on code/modules/projectiles/gun.dm:
```diff
@@ -349,7 +350,7 @@
 	if(gun_light && can_flashlight)
 		for(var/obj/item/flashlight/seclite/S in src)
-			to_chat(user, "<span class='notice'>You unscrew the seclite from [src].</span>")
+			to_chat(user, "<span class='notice'>Вы открутили фонарик от [src.declent_ru(GENITIVE)].</span>")
```
.

larentoun commented on code/modules/projectiles/guns/projectile.dm:
```diff
@@ -152,7 +152,7 @@
 /obj/item/gun/projectile/examine(mob/user)
 	. = ..()
-	. += "Has [get_ammo()] round\s remaining."
+	. += "Боезапаса внутри: [get_ammo()]"
```
.

larentoun commented on code/modules/projectiles/guns/projectile/revolver.dm:
```diff
@@ -85,7 +85,7 @@
 /obj/item/gun/projectile/revolver/examine(mob/user)
 	. = ..()
-	. += "[get_ammo(0,0)] of those are live rounds."
+	. += "Из них заряжено: [get_ammo(0,0)]"
```
.

larentoun commented on code/modules/projectiles/guns/projectile/revolver.dm:
```diff
@@ -191,7 +191,9 @@
-	desc = "A retro high-powered autorevolver typically used by officers of the New Russia military. Uses .357 ammo."	//>10mm hole >.357
+	ru_names = list(NOMINATIVE = "авто-револьвер Уника 6", GENITIVE = "авто-револьвера Уника 6", DATIVE = "авто-револьверу Уника 6", ACCUSATIVE = "авто-револьвер Уника 6", INSTRUMENTAL = "авто-револьвером Уника 6", PREPOSITIONAL = "авто-револьвере Уника 6")
+	gender = MALE
+	desc = "Сильный ретро авторевольвер, который обычно используется офицерами Новорусской армии. Использует патроны калибра .357"	//>10mm hole >.357
```
.

larentoun commented on code/game/objects/items.dm:
```diff
@@ -184,19 +184,19 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
+			size = "огромный"
 		if(WEIGHT_CLASS_GIGANTIC)
-			size = "gigantic"
+			size = "гигантский"
```
.

larentoun commented on code/game/objects/items.dm:
```diff
@@ -184,19 +184,19 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
+			size = "громоздкий"
 		if(WEIGHT_CLASS_HUGE)
-			size = "huge"
+			size = "огромный"
```
.

larentoun commented on code/game/objects/items.dm:
```diff
@@ -184,19 +184,19 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
+			size = "обычного размера"
 		if(WEIGHT_CLASS_BULKY)
-			size = "bulky"
+			size = "громоздкий"
```
.

larentoun commented on code/game/objects/items.dm:
```diff
@@ -184,19 +184,19 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
+			size = "маленький"
 		if(WEIGHT_CLASS_NORMAL)
-			size = "normal-sized"
+			size = "обычного размера"
```
.

VasilevAY replied:
в остальных размер пишется без приставки "... размера", а тут обычного размера
Может просто "обычный"?

larentoun commented on code/game/objects/items.dm:
```diff
@@ -184,19 +184,19 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
+			size = "крохотный"
 		if(WEIGHT_CLASS_SMALL)
-			size = "small"
+			size = "маленький"
```
.

larentoun commented on code/game/objects/items.dm:
```diff
@@ -184,19 +184,19 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
 	switch(src.w_class)
 		if(WEIGHT_CLASS_TINY)
-			size = "tiny"
+			size = "крохотный"
```
.

larentoun commented on code/game/turfs/simulated/floor/plating.dm:
```diff
@@ -359,23 +359,23 @@
 		if(prob(smash_prob))
 			// YAR BE CAUSIN A HULL BREACH
-			visible_message("<span class='danger'>[user] smashes through \the [src] with \the [C]!</span>")
+			visible_message("<span class='danger'>[user.declent_ru(NOMINATIVE)] проламывает [src.declent_ru(ACCUSATIVE)] с помощью [C.declent_ru(GENITIVE)]!</span>")
```
.

larentoun commented on code/game/turfs/simulated/floor/plating.dm:
```diff
@@ -359,23 +359,23 @@
 			smash()
 		else
-			visible_message("<span class='warning'>[user]'s [C.name] bounces against \the [src]!</span>")
+			visible_message("<span class='warning'>[C.declent_ru(NOMINATIVE)] отскакивает от [src.declent_ru(GENITIVE)]!</span>")
```
.

larentoun commented on code/game/turfs/simulated/floor/plating.dm:
```diff
@@ -359,23 +359,23 @@
 	M.do_attack_animation(src)
 	if(M.melee_damage_upper == 0)
-		M.visible_message("<span class='notice'>[M] nudges \the [src].</span>")
+		M.visible_message("<span class='notice'>[M.declent_ru(NOMINATIVE)] подталкивает [src.declent_ru(ACCUSATIVE)].</span>")
```
.

PhantornRU commented on code/datums/mind.dm:
```diff
@@ -144,15 +144,15 @@
 
 	if(LAZYLEN(job_objectives))
-		output += "<HR><B>Job Objectives:</B><UL>"
+		output += "<HR><B>Задачи профессии:</B><UL>"
```
Давай попробуем так Либо же Должностные обязательства. Так звучит получше и по русски.
```suggestion
		output += "<HR><B>Должностные задачи:</B><UL>"
```

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -21,7 +13,7 @@
 	var/germ_level = GERM_LEVEL_AMBIENT // The higher the germ level, the more germ on the atom.
 	var/simulated = TRUE //filter for actions - used by lighting overlays
-	var/atom_say_verb = "says"
+	var/atom_say_verb = "говорит"
```
Вот здесь самое интересное, т.к. должен определяться падеж того кто говорит.
Сказал(а/и)

larentoun replied:
Хм, тогда думаю лучше по умолчанию оставить. Не желательно тут делать функции

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -317,38 +309,34 @@
-			f_name = "a "
 		if(blood_color != "#030303")
-			f_name += "<span class='danger'>blood-stained</span> [name][infix]!"
+			f_name = "<span class='danger'>[genderize_ru(gender, "окровавленный", "окровавленная", "окровавленное", "окровавленные")]</span> [declent_ru(NOMINATIVE)][infix]!"
```
Сначала слово, потом гендерные окончания
Касаемо падежа. Непонятно что он падежит, нужно падежить что-то конкретно, например само это имя.
ОБЯЗАТЕЛЬНО ПРОВЕРЬ, как это происходит. Если что-то ломается, то убираешь отсюда падеж и оставляешь просто [name]
```suggestion
			f_name = "<span class='danger'>окровавленн[genderize_ru(gender, "ый", "ая", "ое", "ые")]</span> [name.declent_ru(NOMINATIVE)][infix]!"
```

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -317,38 +309,34 @@
 		else
-			f_name += "oil-stained [name][infix]."
-	. = list("[bicon(src)] That's [f_name] [suffix]")
+			f_name = "[genderize_ru(gender, "обмасленный", "обмасленная", "обмасленное", "обмасленные")] [declent_ru(NOMINATIVE)][infix]."
```
Сначала слово, потом гендерные окончания

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -317,38 +309,34 @@
 	if(reagents)
 		if(container_type & TRANSPARENT)
-			. += "<span class='notice'>It contains:</span>"
+			. += "<span class='notice'>Контейнер содержит:</span>"
```
Стоит ли обозначать что это контейнер? Это может быть своеобразная емкость. 
Тут на свое усмотрение. Можно в принципе так и так.
```suggestion
			. += "<span class='notice'>Содержание:</span>"
```

larentoun replied:
Подождем мнения остальных, сам не знаю, как лучше.

VasilevAY replied:
а почему нельзя "В контейнере : "?

larentoun replied:
Можно еще как вариант "Внутри:"

PhantornRU replied:
Внутри находятся предметы. А это реагенты.

Нужно использовать содержание, либо емкость. Тоесть:
- Содержит:
- Емкость содержит:

VasilevAY replied:
> Можно еще как вариант "Внутри:"

звучит классно

PhantornRU replied:
Против "Внутри". Нужно делать упор на "Содержание" и/или Емкость. Т.к. это идёт работа именно с реагентами и емкостями, а не всякими коробками.

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -317,38 +309,34 @@
+						. += "<span class='notice'>[reagents.total_volume] единиц различных реагентов.</span>"
 			else
-				. += "<span class='notice'>Nothing.</span>"
+				. += "<span class='notice'>ничего.</span>"
```
```suggestion
				. += "<span class='notice'>Ничего.</span>"
```

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -317,38 +309,34 @@
 		else if(container_type & AMOUNT_VISIBLE)
 			if(reagents.total_volume)
-				. += "<span class='notice'>It has [reagents.total_volume] unit\s left.</span>"
+				. += "<span class='notice'>Контейнер содержит [reagents.total_volume] единиц веществ.</span>"
```
Либо так, либо "емкость". Бутылка и кружка тоже не совсем контейнер, но по прежнему емкость.
```suggestion
				. += "<span class='notice'>Содержит [reagents.total_volume] единиц веществ.</span>"
```

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -317,38 +309,34 @@
+				. += "<span class='notice'>Контейнер содержит [reagents.total_volume] единиц веществ.</span>"
 			else
-				. += "<span class='danger'>It's empty.</span>"
+				. += "<span class='danger'>Контейнер пуст.</span>"
```
Либо "Емкость пуста"
```suggestion
				. += "<span class='danger'>Пустой.</span>"
```

larentoun replied:
Или уточнить, что "Внутри пусто.". Подождем остальных

PhantornRU replied:
Можно так, действительно

VasilevAY replied:
> Или уточнить, что "Внутри пусто.". Подождем остальных

Звучит классно

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -1006,9 +994,9 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 		default_value = ""
 	if(!prompt)
-		prompt = "What would you like the label on [src] to be?"
+		prompt = "Какой ярлык вы хотите поставить на [declent_ru(ACCUSATIVE)]?"
```
Здесь точно всё ок и не нужно уточнять что нужно падежить? Мне кажется или он не видит что здесь нужно падежить?
```suggestion
		prompt = "Какой ярлык вы хотите поставить на [src.declent_ru(ACCUSATIVE)]?"
```

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -1006,9 +994,9 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
+		prompt = "Какой ярлык вы хотите поставить на [declent_ru(ACCUSATIVE)]?"
 
-	var/t = input(user, prompt, "Renaming [src]", default_value)  as text | null
+	var/t = input(user, prompt, "Переименование [declent_ru(GENITIVE)]", default_value)  as text | null
```
```suggestion
	var/t = input(user, prompt, "Переименование [src.declent_ru(GENITIVE)]", default_value)  as text | null
```

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -317,38 +309,34 @@
 /atom/proc/examine(mob/user, infix = "", suffix = "")
 	//This reformat names to get a/an properly working on item descriptions when they are bloody
-	var/f_name = "\a [src][infix]."
+	var/f_name = "[declent_ru(NOMINATIVE)][infix]."
```
Проверь этот момент, точно ли он падежит всё правильно? Мне кажется лучше уточнить ему, что падежить нужно сурс.
```suggestion
	var/f_name = "[src.declent_ru(NOMINATIVE)][infix]."
```

larentoun replied:
Стоит, да. И подежит нормально

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -1017,13 +1005,13 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 		return null
 	else if(implement && implement.loc != user)
-		to_chat(user, "<span class='warning'>You no longer have the pen to rename [src].</span>")
+		to_chat(user, "<span class='warning'>У вас нет ручки, чтобы переименовать [declent_ru(ACCUSATIVE)].</span>")
```
```suggestion
		to_chat(user, "<span class='warning'>У вас нет ручки, чтобы переименовать [src.declent_ru(ACCUSATIVE)].</span>")
```

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -1017,13 +1005,13 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 		return null
 	else if(!in_range(src, user))
-		to_chat(user, "<span class='warning'>You cannot rename [src] from here.</span>")
+		to_chat(user, "<span class='warning'>Вы не можете переименовать [declent_ru(ACCUSATIVE)] отсюда.</span>")
```
Указать что именно падежит.

PhantornRU commented on code/game/atoms.dm:
```diff
@@ -1017,13 +1005,13 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 		return null
 	else if (user.incapacitated(ignore_lying = TRUE))
-		to_chat(user, "<span class='warning'>You cannot rename [src] in your current state.</span>")
+		to_chat(user, "<span class='warning'>Вы не можете переименовать [declent_ru(ACCUSATIVE)] в вашем текущем состоянии.</span>")
```
Указать что именно падежит.

PhantornRU commented on code/game/atoms_movable.dm:
```diff
@@ -120,13 +120,13 @@
 	if(force < (move_resist * MOVE_FORCE_PULL_RATIO))
 		if(show_message)
-			to_chat(user, "<span class='warning'>[src.name] слишком тяжелый!</span>")
+			to_chat(user, "<span class='warning'>[src.declent_ru(NOMINATIVE)] слишком тяжелый!</span>")
```
Да, знаю что это был когда-то мой старый перевод.
Тут тоже проверь, всё ли будет ок по падежу, т.к. предметы/объекты, да и в целом Атомы, если не ошибаюсь тоже имеют пол.
```suggestion
			to_chat(user, "<span class='warning'>[src.declent_ru(NOMINATIVE)] слишком тяжел[genderize_ru(src.gender,"ый","ая","ое","ые")]!</span>")
```

PhantornRU commented on code/game/gamemodes/vampire/vampire.dm:
```diff
@@ -399,7 +399,7 @@
 	slaved.leave_serv_hud(vampire_mind)
 	update_vampire_icons_removed(vampire_mind)
-	vampire_mind.current.visible_message("<span class='userdanger'>Кажется, будто тяжёлый груз упал с плеч [vampire_mind.current]!</span>", "<span class='userdanger'>Тёмная пелена спала с вашего рассудка. Ваш разум прояснился. Вы больше не [usr.gender == MALE ? "раб" : "раба"] вампира и снова отвечаете за свои действия!</span>")
+	vampire_mind.current.visible_message("<span class='userdanger'>Кажется, будто тяжёлый груз упал с плеч [vampire_mind.current]!</span>", "<span class='userdanger'>Тёмная пелена спала с вашего рассудка. Ваш разум прояснился. Вы больше не [genderize_ru(vampire_mind.current.gender, "раб", "рабыня", "раб", "рабы")] вампира и снова отвечаете за свои действия!</span>")
```
вынести слово отдельно от гендеризации

PhantornRU commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -238,7 +238,7 @@
 
 /obj/effect/proc_holder/spell/vampire/self/shapeshift/cast(list/targets, mob/user = usr)
-	user.visible_message("<span class='warning'>[user] transforms!</span>")
+	user.visible_message("<span class='warning'>Внешность [user.declent_ru(GENITIVE)] изменяется!</span>")
```
```suggestion
	user.visible_message("<span class='warning'>[user.declent_ru(GENITIVE)] преображается!</span>")
```

PhantornRU commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -293,7 +293,7 @@
 /obj/effect/proc_holder/spell/vampire/targetted/enthrall/cast(list/targets, mob/user = usr)
 	for(var/mob/living/target in targets)
-		user.visible_message("<span class='warning'>[user] кусает [target] в шею!</span>", "<span class='warning'>Вы кусаете [target] в шею и начинаете передачу части своей силы.</span>")
+		user.visible_message("<span class='warning'>[user.declent_ru(NOMINATIVE)] кусает [target.declent_ru(ACCUSATIVE)] в шею!</span>", "<span class='warning'>Вы кусаете [target.declent_ru(ACCUSATIVE)] в шею и начинаете передачу части своей силы.</span>")
```
Нужна гендеризация глаголов

PhantornRU commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -319,13 +319,13 @@
 		return 0
 	if(!C.mind)
-		to_chat(user, "<span class='warning'>Разум [C.name] сейчас не здесь, поэтому порабощение не удастся.</span>")
+		to_chat(user, "<span class='warning'>Разум [C.declent_ru(GENITIVE)] сейчас не здесь, поэтому порабощение не удастся.</span>")
```
А может так переведем, кстати?
```suggestion
		to_chat(user, "<span class='warning'>Разум [C.declent_ru(GENITIVE)] где-то витает, поэтому порабощение не удастся.</span>")
```

larentoun replied:
Тоже можно, подождем остальных

PhantornRU replied:
Уж точно не оставлять "сейчас не здесь". Это очень плохое словосочетание.

VasilevAY replied:
а напомните контекст, что происходит?

larentoun replied:
Во время траллирования цель была в ССД/в гостах

PhantornRU replied:
игрок покинул тело (админ ф6 или кататоник)

VasilevAY replied:
ага
Предлагаю так
"Душа name покинуло тело, поэтому порабощение не удастся"

VasilevAY replied:
> 

кстати, говно идея. Нужен падеж на это.

PhantornRU replied:
покинула*

Душа - женский род

larentoun replied:
Падежы не проблема - скоро сделаю систему для падежей имен персонажей

larentoun replied:
Тогда лучше вариант с "Душа покинула"?

PhantornRU commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -319,13 +319,13 @@
 		return 0
 	if(enthrall_safe || ( C.mind in SSticker.mode.vampires )||( C.mind.vampire )||( C.mind in SSticker.mode.vampire_enthralled ))
-		C.visible_message("<span class='warning'>Похоже что [C] сопротивляется захвату!</span>", "<span class='notice'>Вы ощущаете в голове знакомое ощущение, но оно быстро проходит.</span>")
+		C.visible_message("<span class='warning'>Похоже что [C.declent_ru(NOMINATIVE)] сопротивляется захвату!</span>", "<span class='notice'>Вы ощущаете в голове знакомое ощущение, но оно быстро проходит.</span>")
```
Нужна гендеризация глаголов - сопротивля**[ЕТ/ЮТ]**ся.
Дионы например имеют множественный пол

PhantornRU commented on code/game/gamemodes/vampire/vampire_powers.dm:
```diff
@@ -319,13 +319,13 @@
 		return 0
 	if(!affects(C))
-		C.visible_message("<span class='warning'>Похоже что [C] сопротивляется захвату!</span>", "<span class='notice'>Вера в [SSticker.Bible_deity_name] защищает ваш разум от всякого зла.</span>")
+		C.visible_message("<span class='warning'>Похоже что [C.declent_ru(NOMINATIVE)] сопротивляется захвату!</span>", "<span class='notice'>Вера в [SSticker.Bible_deity_name] защищает ваш разум от всякого зла.</span>")
```
Аналогично

PhantornRU commented on code/game/machinery/Freezer.dm:
```diff
@@ -163,7 +164,8 @@
 /obj/machinery/atmospherics/unary/heat_reservoir/heater/
-	name = "нагреватель"
+	name = "heater"
+	ru_names = list(NOMINATIVE = "нагреватель", GENITIVE = "нагревателя", DATIVE = "нагревателю", ACCUSATIVE = "нагреватель", INSTRUMENTAL = "нагревателем", PREPOSITIONAL = "нагревателе")
```
Может "Обогреватель"? Его чаще используют

PhantornRU commented on code/game/objects/items/weapons/dice.dm:
```diff
@@ -27,11 +28,12 @@
 
 /obj/item/storage/pill_bottle/dice/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] Игра[pluralize_ru(user.gender,"ет","ют")] со смертью! Похоже, он[genderize_ru(user.gender,"","а","о","и")] пыта[pluralize_ru(user.gender,"ется","ются")] покончить жизнь самоубийством!</span>")
+	user.visible_message("<span class='suicide'>[user] Игра[pluralize_ru(user.gender,"ет","ют")] со смертью! Похоже [genderize_ru(user.gender, "он пытается", "она пытается", "оно пытается", "они пытаются")] покончить жизнь самоубийством!</span>")
```
Не стоит мешать всё в одну кучу
```suggestion
	user.visible_message("<span class='suicide'>[user] Игра[pluralize_ru(user.gender,"ет","ют")] со смертью! Похоже, он[genderize_ru(user.gender,"","а","о","и")] пыта[pluralize_ru(user.gender,"ется","ются")] покончить жизнь самоубийством!</span>")

```

PhantornRU commented on code/game/objects/items/weapons/dice.dm:
```diff
@@ -51,7 +53,7 @@
 
 /obj/item/dice/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] играет со смертью! Похоже [user.p_theyre()] пытается покончить жизнь самоубийством!</span>")
+	user.visible_message("<span class='suicide'>[user] играет со смертью! Похоже [genderize_ru(user.gender, "он пытается", "она пытается", "оно пытается", "они пытаются")] покончить жизнь самоубийством!</span>")
```
опять всё в куче

PhantornRU commented on code/game/verbs/suicide.dm:
```diff
@@ -107,7 +107,7 @@
 					return
 
-		to_chat(viewers(src), "<span class='danger'>[src] [replacetext(pick(dna.species.suicide_messages), "their", p_their())] It looks like [p_theyre()] trying to commit suicide.</span>")
+		to_chat(viewers(src), "<span class='danger'>[src.declent_ru(NOMINATIVE)] [pick(dna.species.suicide_messages)] Похоже, что [genderize_ru(gender, "он хочет", "она хочет", "оно хочет", "они хотят")] совершить самоубийство.</span>")
```
опять куча мала

PhantornRU commented on code/modules/food_and_drinks/drinks/drinks.dm:
```diff
@@ -112,16 +112,16 @@
+			. += "<span class='notice'> \The [src.declent_ru(NOMINATIVE)] почти пуст!</span>"
 		else if(reagents.total_volume <= volume*0.66)
-			. += "<span class='notice'> \The [src] is half full!</span>"// We're all optimistic, right?!
+			. += "<span class='notice'> \The [src.declent_ru(NOMINATIVE)] наполовину наполон!</span>"// We're all optimistic, right?!
```
```suggestion
			. += "<span class='notice'> \The [src.declent_ru(NOMINATIVE)] наполовину полон!</span>"// We're all optimistic, right?!
```

PhantornRU commented on code/modules/mob/living/carbon/carbon.dm:
```diff
@@ -305,28 +305,28 @@
 
 		if(brutedamage > 0)
-			status = "bruised"
+			status = "имеет ушибы"
```
отсутствует гендеризация.
"Помят" будет звучать получше. Тоже гендеризировать нужно

PhantornRU commented on code/modules/mob/living/carbon/carbon.dm:
```diff
@@ -1053,21 +1053,21 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 	else
 		if(toEat.junkiness && satiety < -150 && nutrition > NUTRITION_LEVEL_STARVING + 50 )
-			to_chat(src, "<span class='notice'>You don't feel like eating any more junk food at the moment.</span>")
+			to_chat(src, "<span class='notice'>Вы пока что не хотите есть еще больше нездоровой пищи.</span>")
```
```suggestion
			to_chat(src, "<span class='notice'>Вам больше не хочется есть вредную пищу.</span>")
```

PhantornRU commented on code/modules/mob/living/carbon/carbon.dm:
```diff
@@ -1053,21 +1053,21 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
+			to_chat(src, "<span class='notice'>Вы откусываете [toEat.declent_ru(ACCUSATIVE)].</span>")
 		else if(fullness > 500 && fullness < 600)
-			to_chat(src, "<span class='notice'>You unwillingly chew a bit of [toEat].</span>")
+			to_chat(src, "<span class='notice'>Вы не хотя берете кусок [toEat.declent_ru(GENITIVE)].</span>")
```
```suggestion
			to_chat(src, "<span class='notice'>Вы неохотно прожевываете [toEat.declent_ru(GENITIVE)].</span>")
```

PhantornRU commented on code/modules/mob/living/carbon/carbon.dm:
```diff
@@ -1053,21 +1053,21 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
+			to_chat(src, "<span class='notice'>Вы не хотя берете кусок [toEat.declent_ru(GENITIVE)].</span>")
 		else if(fullness > (600 * (1 + overeatduration / 2000)))	// The more you eat - the more you can eat
-			to_chat(src, "<span class='warning'>You cannot force any more of [toEat] to go down your throat.</span>")
+			to_chat(src, "<span class='warning'>Вы больше не можете заставить себя проглотить [toEat.declent_ru(ACCUSATIVE)].</span>")
```
```suggestion
			to_chat(src, "<span class='warning'>В вас больше не лезет [toEat.declent_ru(ACCUSATIVE)].</span>")
```

PhantornRU commented on code/modules/mob/living/carbon/carbon.dm:
```diff
@@ -1077,15 +1077,15 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 	if(ispill(toEat) || fullness <= (600 * (1 + overeatduration / 1000)))
 		if(!toEat.instant_application)
-			visible_message("<span class='warning'>[user] attempts to force [src] to [toEat.apply_method] [toEat].</span>")
+			visible_message("<span class='warning'>[user.declent_ru(NOMINATIVE)] пытается насильно [src.declent_ru(ACCUSATIVE)] [toEat.apply_method] [toEat.declent_ru(ACCUSATIVE)].</span>")
```
ет/ют
[pluralize_ru(src.gender,"ет","ют")]

PhantornRU replied:
[pluralize_ru(user.gender,"ет","ют")]

PhantornRU commented on code/modules/mob/living/carbon/carbon.dm:
```diff
@@ -1077,15 +1077,15 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
+			visible_message("<span class='warning'>[user.declent_ru(NOMINATIVE)] пытается насильно [src.declent_ru(ACCUSATIVE)] [toEat.apply_method] [toEat.declent_ru(ACCUSATIVE)].</span>")
 	else
-		visible_message("<span class='warning'>[user] cannot force anymore of [toEat] down [src]'s throat.</span>")
+		visible_message("<span class='warning'>[user.declent_ru(NOMINATIVE)] больше не может насильно заставить [src.declent_ru(ACCUSATIVE)] проглотить [toEat.declent_ru(ACCUSATIVE)].</span>")
```
может/могут

PhantornRU commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -568,8 +568,8 @@
-			H.visible_message("<span class='danger'>[P.name] тонет в песчаном теле [H] без видимого вреда здоровью!</span>", \
-			"<span class='userdanger'>[P.name] тонет в песчаном теле [H] без видимого вреда здоровью!</span>")
+			H.visible_message("<span class='danger'>[P.declent_ru(NOMINATIVE)] тонет в песчаном теле [H.declent_ru(GENITIVE)] без видимого вреда здоровью!</span>", \
+			"<span class='userdanger'>[P.declent_ru(NOMINATIVE)] тонет в песчаном теле [H.declent_ru(GENITIVE)] без видимого вреда здоровью!</span>")
```
ет/ют

PhantornRU commented on code/modules/mob/living/carbon/human/species/grey.dm:
```diff
@@ -43,11 +43,11 @@
 	if(method == REAGENT_TOUCH)
 		if(H.wear_mask)
-			to_chat(H, "<span class='danger'>Ваша [H.wear_mask] защищает вас от кислоты!</span>")
+			to_chat(H, "<span class='danger'>[genderize_ru(H.wear_mask.gender, "Ваш", "Ваша", "Ваше", "Ваши")] [H.wear_mask.declent_ru(NOMINATIVE)] защищает вас от кислоты!</span>")
```
отделить ВАШ от гендеризации

PhantornRU commented on code/modules/mob/living/carbon/human/species/grey.dm:
```diff
@@ -43,11 +43,11 @@
 
 		if(H.head)
-			to_chat(H, "<span class='danger'>Ваша [H.wear_mask] защищает вас от кислоты!</span>")
+			to_chat(H, "<span class='danger'>[genderize_ru(H.wear_mask.gender, "Ваш", "Ваша", "Ваше", "Ваши")] [H.wear_mask.declent_ru(NOMINATIVE)] защищает вас от кислоты!</span>")
```
отделить слово от гендеризации

PhantornRU commented on code/modules/mob/living/carbon/human/species/skeleton.dm:
```diff
@@ -54,7 +54,7 @@
 				if(L.mend_fracture())
 					break // We're only checking one limb here, bucko
-		if(prob(25)) //25% шанс на случайную шутливую фразу
+		if(prob(3)) //3% шанс на случайную шутливую фразу
```
Правильно фикс, а то Мистер Скелетал из-за моей правки всех задалбывать начал)

larentoun replied:
:D

PhantornRU commented on code/modules/mob/living/simple_animal/hostile/killertomato.dm:
```diff
@@ -15,7 +15,7 @@
 	melee_damage_lower = 8
 	melee_damage_upper = 12
-	attacktext = "кусает"
+	attacktext = "ударяет"
```
Это гребаный томат убийца. У него зубы с половину его тела :)))
```suggestion
	attacktext = "вгрызается"
```

larentoun replied:
По звуку, видимо, томат всей своей массой таранит)

PhantornRU replied:
Звук - это заглушка. Этот томат злой и беспощадный. Он точно не банально бьет.

PhantornRU commented on code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm:
```diff
@@ -39,7 +39,7 @@ Difficulty: Hard
 	health = 2500
 	maxHealth = 2500
-	attacktext = "качает"
+	attacktext = "ударяет"
```
Иерофант, проникает своей волной
```suggestion
	attacktext = "проникает"
```

larentoun replied:
По звуку, видимо, он сам собой бьет. Он же палка! Надо в игре глянуть

PhantornRU replied:
Не по звуку думай, а по логике.

PhantornRU commented on code/modules/mob/living/simple_animal/hostile/netherworld.dm:
```diff
@@ -25,7 +25,7 @@
 	icon_living = "mi-go"
 	icon_dead = "mi-go-dead"
-	attacktext = "мучает"
+	attacktext = "разрывает"
```
Миго заставляет твою голову ныть, он именно что мучает. Это создание сотоны
```suggestion
	attacktext = "мучает"
```

larentoun replied:
А, вот оно как

PhantornRU commented on code/modules/mob/living/simple_animal/hostile/retaliate/kangaroo.dm:
```diff
@@ -15,7 +15,7 @@
 	melee_damage_lower = 5 // avg damage 12.5 without kick, (12.5+12.5+60)/3=25 with kick
 	melee_damage_upper = 20
-	attacktext = "пинает"
+	attacktext = "полоснул"
```
Это кенгуру, с чего он полощет, а не пинает?
```suggestion
	attacktext = "пинает"
```

larentoun replied:
По звуку и комментарию - у кенгуру очень острые когти тут :D

PhantornRU replied:
Not sound. Use logic

Вообще, лапами не только синяки можно поставить с размаха, но и надрезать. + "пинок" был изначально.

PhantornRU commented on code/modules/mob/living/simple_animal/shade.dm:
```diff
@@ -15,7 +15,7 @@
 	melee_damage_lower = 5
 	melee_damage_upper = 15
-	attacktext = "опустошает"
+	attacktext = "высасывает жизнь из"
```
Это и есть опустошает
```suggestion
	attacktext = "опустошает"
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -89,15 +89,15 @@
+		. += "<span class='notice'>Нажмите Альт-клик, чтобы сменить раскраску.</span>"
 	if(unique_rename)
-		. += "<span class='notice'>Use a pen on it to rename it.</span>"
+		. += "<span class='notice'>Используйте ручку, чтобы переименовать оружие.</span>"
```
```suggestion
		. += "<span class='notice'>Воспользуйтесь ручкой, для переименования оружия.</span>"
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -89,15 +89,15 @@
+		. += "<span class='notice'>Используйте ручку, чтобы переименовать оружие.</span>"
 	if(bayonet)
-		. += "It has \a [bayonet] [can_bayonet ? "" : "permanently "]affixed to it."
+		. += "Оружие имеет [can_bayonet ? "" : "перманентно "]закрепленн[genderize_ru(bayonet.gender, "ый", "ую", "ое", "ые")] [bayonet.declent_ru(ACCUSATIVE)]."
```
```suggestion
		. += "У оружия [can_bayonet ? "закрепленн[genderize_ru(bayonet.gender, "ый", "ую", "ое", "ые")]" : "встроен[genderize_ru(bayonet.gender, "ый", "ая", "ое", "ые")]"] [bayonet.declent_ru(ACCUSATIVE)]."
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -89,15 +89,15 @@
+		. += "Оружие имеет [can_bayonet ? "" : "перманентно "]закрепленн[genderize_ru(bayonet.gender, "ый", "ую", "ое", "ые")] [bayonet.declent_ru(ACCUSATIVE)]."
 		if(can_bayonet) //if it has a bayonet and this is false, the bayonet is permanent.
-			. += "<span class='info'>[bayonet] looks like it can be <b>unscrewed</b> from [src].</span>"
+			. += "<span class='info'>Похоже, что [bayonet.declent_ru(ACCUSATIVE)] можно снять отверткой.</span>"
```
```suggestion
			. += "<span class='info'>Похоже [bayonet.declent_ru(ACCUSATIVE)] можно снять отверткой.</span>"
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -89,15 +89,15 @@
+			. += "<span class='info'>Похоже, что [bayonet.declent_ru(ACCUSATIVE)] можно снять отверткой.</span>"
 	else if(can_bayonet)
-		. += "It has a <b>bayonet</b> lug on it."
+		. += "На оружии есть крепление для <b>байонеты</b>."
```
```suggestion
		. += "На оружии есть крепление для <b>штыка</b>."
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -127,9 +127,9 @@
 		if(message)
 			if(pointblank)
-				user.visible_message("<span class='danger'>[user] fires [src] point blank at [target]!</span>", "<span class='danger'>You fire [src] point blank at [target]!</span>", "<span class='italics'>You hear \a [fire_sound_text]!</span>")
+				user.visible_message("<span class='danger'>[user.declent_ru(NOMINATIVE)] стреляет из [src.declent_ru(GENITIVE)] в упор по [target.declent_ru(DATIVE)]!</span>", "<span class='danger'>Вы стреляете из [src.declent_ru(GENITIVE)] в упор по [target.declent_ru(DATIVE)]!</span>", "<span class='italics'>Вы слышите [fire_sound_text]!</span>")
```
ет/ют. Не забываем про дион с пулеметами

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -127,9 +127,9 @@
+				user.visible_message("<span class='danger'>[user.declent_ru(NOMINATIVE)] стреляет из [src.declent_ru(GENITIVE)] в упор по [target.declent_ru(DATIVE)]!</span>", "<span class='danger'>Вы стреляете из [src.declent_ru(GENITIVE)] в упор по [target.declent_ru(DATIVE)]!</span>", "<span class='italics'>Вы слышите [fire_sound_text]!</span>")
 			else
-				user.visible_message("<span class='danger'>[user] fires [src]!</span>", "<span class='danger'>You fire [src]!</span>", "You hear \a [fire_sound_text]!")
+				user.visible_message("<span class='danger'>[user.declent_ru(NOMINATIVE)] стреляет из [src.declent_ru(GENITIVE)]!</span>", "<span class='danger'>Вы стреляете из [src.declent_ru(GENITIVE)]!</span>", "Вы слышите [fire_sound_text]!")
```
ет/ют

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -173,14 +173,14 @@
 
 	if(weapon_weight == WEAPON_HEAVY && user.get_inactive_hand())
-		to_chat(user, "<span class='userdanger'>You need both hands free to fire \the [src]!</span>")
+		to_chat(user, "<span class='userdanger'>Вам нужна вторая свободная рука для стрельбы из [src.declent_ru(GENITIVE)]!</span>")
```
```suggestion
		to_chat(user, "<span class='userdanger'>Для стрельбы из [src.declent_ru(GENITIVE)] нужны свободные руки!</span>")
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -202,7 +202,7 @@
 		return 0
 	if(restricted_species && restricted_species.len && !is_type_in_list(user.dna.species, restricted_species))
-		to_chat(user, "<span class='danger'>[src] is incompatible with your biology!</span>")
+		to_chat(user, "<span class='danger'>[src.declent_ru(NOMINATIVE)] несовместим[genderize_ru(src, "", "а", "о", "и")] с вашей биологией!</span>")
```
```suggestion
		to_chat(user, "<span class='danger'>[src.declent_ru(NOMINATIVE)] несовместим[genderize_ru(src, "", "а", "о", "ы")] с вашей биологией!</span>")
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -226,7 +226,7 @@
 		if(chambered && chambered.harmful)
 			if(HAS_TRAIT(user, TRAIT_PACIFISM)) // If the user has the pacifist trait, then they won't be able to fire [src] if the round chambered inside of [src] is lethal.
-				to_chat(user, "<span class='warning'>[src] is lethally chambered! You don't want to risk harming anyone...</span>")
+				to_chat(user, "<span class='warning'>[src.declent_ru(NOMINATIVE)] заряжен летальным патроном! Вы не хотите никому случайно навредить...</span>")
```
```suggestion
				to_chat(user, "<span class='warning'>[src.declent_ru(NOMINATIVE)] заряжен летальным патроном! Вам не хочется никому навредить...</span>")
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -259,7 +259,7 @@
 			if(HAS_TRAIT(user, TRAIT_PACIFISM)) // If the user has the pacifist trait, then they won't be able to fire [src] if the round chambered inside of [src] is lethal.
 				if(chambered.harmful) // Is the bullet chambered harmful?
-					to_chat(user, "<span class='warning'>[src] is lethally chambered! You don't want to risk harming anyone...</span>")
+					to_chat(user, "<span class='warning'>[src.declent_ru(NOMINATIVE)] заряжен летальным патроном! Вы не хотите никому случайно навредить...</span>")
```
```suggestion
					to_chat(user, "<span class='warning'>[src.declent_ru(NOMINATIVE)] заряжен летальным патроном! Вам не хочется никому навредить...</span>")
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -307,7 +307,7 @@
 				if(!user.unEquip(I))
 					return
-				to_chat(user, "<span class='notice'>You click [S] into place on [src].</span>")
+				to_chat(user, "<span class='notice'>Вы присоединили [S.declent_ru(ACCUSATIVE)] на [src.declent_ru(ACCUSATIVE)].</span>")
```
```suggestion
				to_chat(user, "<span class='notice'>Вы присоединили [S.declent_ru(ACCUSATIVE)] к [src.declent_ru(ACCUSATIVE)].</span>")
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -322,15 +322,16 @@
 			var/t = rename_interactive(user, I, use_prefix = FALSE)
 			if(!isnull(t))
-				to_chat(user, "<span class='notice'>You name the gun [name]. Say hello to your new friend.</span>")
+				to_chat(user, "<span class='notice'>Вы назвали оружие - [name]. Скажи привет своему новому другу.</span>")
```
Никто не говорит "скажи ему привет"
```suggestion
				to_chat(user, "<span class='notice'>Вы назвали оружие - [name]. Передай привет своему новому другу.</span>")
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -322,15 +322,16 @@
 			return
 		K.forceMove(src)
-		to_chat(user, "<span class='notice'>You attach [K] to [src]'s bayonet lug.</span>")
+		to_chat(user, "<span class='notice'>Вы закрепили [K.declent_ru(ACCUSATIVE)] на крепление для байонетты у [src.declent_ru(GENITIVE)].</span>")
```
Закрепили на креплении. Масло масляное
```suggestion
		to_chat(user, "<span class='notice'>Вы вставили [K.declent_ru(ACCUSATIVE)] на крепление штыка у [src.declent_ru(GENITIVE)].</span>")
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -404,7 +405,7 @@
 	if(gun_light.on)
 		toggle_gunlight()
-		visible_message("<span class='danger'>[src]'s light fades and turns off.</span>")
+		visible_message("<span class='danger'>Свет от [src.declent_ru(GENITIVE)] тухнет и отключается.</span>")
```
```suggestion
		visible_message("<span class='danger'>Свет [src.declent_ru(GENITIVE)] тухнет и погасает.</span>")
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -415,19 +416,19 @@
 
 /obj/item/gun/proc/reskin_gun(mob/M)
-	var/choice = input(M,"Warning, you can only reskin your weapon once!","Reskin Gun") in options
+	var/choice = input(M,"Внимание! Вы можете поменять раскраску только один раз!","Перекраска оружия") in options
```
```suggestion
	var/choice = input(M,"Внимание! Вы можете перекрасить оружие только один раз!","Перекраска оружия") in options
```

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -415,19 +416,19 @@
 			return
 		current_skin = options[choice]
-		to_chat(M, "Your gun is now skinned as [choice]. Say hello to your new friend.")
+		to_chat(M, "У вашего оружия сейчас стоит раскраска [choice]. Скажи привет своему новому другу.")
```
```suggestion
		to_chat(M, "У вашего оружия сейчас раскраска [choice]. Передай привет своему новому другу.")
```

PhantornRU replied:
Но честно, я бы "новому" заменил на "Обновленному".

larentoun replied:
Тут вероятно больше фокус на то, что именно сейчас пушка стала другом, с именем

PhantornRU replied:
значит оставляем "новому другу"

PhantornRU commented on code/modules/projectiles/gun.dm:
```diff
@@ -438,26 +439,26 @@
 		if(user)
 			if(user == target)
-				user.visible_message("<span class='notice'>[user] decided life was worth living.</span>")
+				user.visible_message("<span class='notice'>[user.declent_ru(NOMINATIVE)] решает, что жизнь того стоит.</span>")
```
Думаю для более лучшего словца и против косоязычия:
```suggestion
				user.visible_message("<span class='notice'>[user.declent_ru(NOMINATIVE)] решает, что жизнь не так безнадежна.</span>")
```

PhantornRU commented on code/modules/projectiles/guns/projectile/revolver.dm:
```diff
@@ -31,7 +31,7 @@
 	var/num_loaded = magazine.attackby(A, user, params, 1)
 	if(num_loaded)
-		to_chat(user, "<span class='notice'>You load [num_loaded] shell\s into \the [src].</span>")
+		to_chat(user, "<span class='notice'>Вы заряжаете [num_loaded] единиц боезапаса в [src.declent_ru(ACCUSATIVE)].</span>")
```
единиц боезопаса лучше заменить на "патрон[/ов]"

larentoun replied:
1 патрон, 2-4 патрона, 5+ патронов. Неужели проверку пихать на последнюю цифру? Нужно что-то придумать

VasilevAY replied:
вариант, когда нужно еще падежи учитывать - плохо, надо подумать, согласен

PhantornRU replied:
пока можно оставить так как есть и оставить мозгоебку для потомков. СБ-шники переводчики думаю когда-нибудь да поправят как им самим хочется и напишут падежные системы.

PhantornRU commented on code/modules/projectiles/guns/projectile/revolver.dm:
```diff
@@ -49,9 +49,9 @@
 			num_unloaded++
 	if(num_unloaded)
-		to_chat(user, "<span class='notice'>You unload [num_unloaded] shell\s from [src].</span>")
+		to_chat(user, "<span class='notice'>Вы разряжаете [num_unloaded] единиц боезапаса из [src.declent_ru(GENITIVE)].</span>")
```
тоже самое

larentoun replied:
ответ выше

PhantornRU commented on code/modules/reagents/chemistry/machinery/pandemic.dm:
```diff
@@ -199,7 +199,7 @@
 		printing = 1
 		var/obj/item/paper/P = new /obj/item/paper(loc)
-		visible_message("<span class='notice'>[src] гремит и печатает лист бумаги.</span>")
+		visible_message("<span class='notice'>[src.declent_ru(NOMINATIVE)] гремит и печатает лист бумаги.</span>")
```
ит/ят
ет/ют

PhantornRU reviewed (changes requested):
Проверил перевод, убил пару часов. Жду исправлений

PhantornRU commented:
Мог какие-то моменты пропустить. Но если ты какие-то слова в кучу смешивал и я на это не указал, или ет/ют пропустил, не указал что именно нужно гендеризировать - это тоже самостоятельно поправь

VasilevAY commented on code/datums/mind.dm:
```diff
@@ -134,7 +134,7 @@
 	if(!recipient)
 		recipient = current
-	var/output = {"<meta charset="UTF-8"><B>[current.real_name]'s Memories:</B><HR>"}
+	var/output = {"<meta charset="UTF-8"><B>Память [current.real_name]:</B><HR>"}
```
Здесь лучше "Воспоминания", как мне кажется

PhantornRU replied:
Согласен, так будет лучше

VasilevAY commented on code/datums/mind.dm:
```diff
@@ -144,15 +144,15 @@
 
 	if(LAZYLEN(job_objectives))
-		output += "<HR><B>Job Objectives:</B><UL>"
+		output += "<HR><B>Должностные задачи:</B><UL>"
```
Еще лучше звучит "должностные инструкции"

PhantornRU replied:
Инструкции здесь излишни и не подходят по смыслу

larentoun commented on code/modules/mob/living/carbon/carbon.dm:
```diff
@@ -305,28 +305,28 @@
-			status += "numb"
+			status += "онемел[genderize_ru(LB.gender, "", "а", "о", "и")]"
 		if(LB.status & ORGAN_MUTATED)
 			status = "weirdly shapen."
```
.

larentoun committed:
fix: мелкие правки перевода

fix: правки перевода

fix: правки перевода

fix: правки перевода

fix: мелкие правки

fix: мелкие правки перевода

fix: мелкие фиксы перевода

fix: мелкие правки перевода для установки

PhantornRU commented on code/datums/mind.dm:
```diff
@@ -144,15 +144,15 @@
 		var/obj_count = 1
 		for(var/datum/job_objective/objective in job_objectives)
-			output += "<LI><B>Task #[obj_count]</B>: [objective.get_description()]</LI>"
+			output += "<LI><B>Задача #[obj_count]</B>: [objective.get_description("ru")]</LI>"
```
Хммм, а может как "запрос" сделать?

PhantornRU replied:
Да, давай тогда лучше на "запрос" поменяем. Это гораздо логичнее и не будет сливаться с другими задачами.

larentoun committed:
fix: Учет падежей с закрытым лицом

правка

larentoun commented:
Надо новый коммит проверить. Я вроде потыкала - работает
